src/aura/BGE_TemplateDetails/BGE_TemplateDetailsController.js

```diff
@@ -0,0 +1,98 @@
+({ // eslint-disable-line
```

Comment:
I'm guessing it's because of this lint rule: https://eslint.org/docs/rules/no-extra-parens ...since the parens are expected for lightning components, another option would be to configure eslint to disable that rule.